Controllers/AdminController.php

```diff
@@ -18,6 +18,9 @@ public function __construct()
    */
   public function adminPageAction()
   {
+    if ($this->userStateClass === "user-logged-out") {
```

Comment:
Everything from admin controller should redirect to login, that means you should add logic in constructor.
Also, condition is not really ideal. Please use global $user declaration and check if $user is NULL or not.